### DISCUSSION TITLE
import { config } from 'md-editor-v3'   This is global configuration. How can I handle the configuration for individual instances?

### DISCUSSION THREAD

**Discussion by shaotanliang** [imzbf/md-editor-v3#527] [Q&A]
This config will affect all instances within the same project. Is there any other way to avoid this issue?

**Comment by imzbf** [comment]
For some configuration items that support differentiation, the editor's `editorId` value is provided in the input parameters, for example: [markdownItConfig](https://imzbf.github.io/md-editor-v3/en-US/docs#%F0%9F%8D%A4%20markdownItConfig), and static configuration items are impossible to distinguish by default

**Reply by shaotanliang** [reply]
Will we consider distinguishing static configuration item functions ？

**Reply by imzbf** [reply]
Sorry, we won't do that
